app/views/spaces/show.html.erb

```diff
@@ -1,2 +1,15 @@
 <h1>Spaces#show</h1>
 <p>Find me in app/views/spaces/show.html.erb</p>
+
+<ul class="list-group">
+  <% spaces.each do |space| %>
+    <li class="list-group-item">
+      <%= space.location %>
+      <%= space.description %>
+      <%= space.volume %>
+      <%= space.price_per_month %>
+      </li>
+  <% end %>
+
+<%= link_to "book this space", new_space_reservation_path(@space) %>
+<%= link_to "back to all spaces", spaces_path %>
```

Comment:
I'd likely capitalize the first word for and possibly rename it to the action, like so "View all spaces"